dts/bindings/battery/battery.yaml

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright 2022 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+   Battery parameters and info
+
+   Voltage is in millivolts
+   Current is in milliamperes
+   Temperature is in Celsius degrees
```

Comment:
Does this need to be specified for every driver? It's part of the API, I think it makes sense to have unit markers in the enum comment and probably even in the properties below, then this should be redundant.

Reply:
So looking closer at this, I think the battery.yaml ought to be clearly defined in a later PR. I'm keeping this PR to defining a new fuel-gauge API with a sample driver and basic tests (which I'm happy to move to another PR, if other reviewers are happy with that).